docs/reference/editor-settings.md

```diff
@@ -256,6 +256,66 @@ like code completion, hover, go to definition, etc.
 
 ______________________________________________________________________
 
+## `showSyntaxErrors`
+
+Whether to show syntax error diagnostics.
+
+This is useful when using ty with other language servers, allowing the user to refer to syntax errors
+from only one source.
+
+**Default value**: `true`
+
+**Type**: `bool`
+
+**Example usage**:
+
+=== "VS Code"
+
+    ```json
+    {
+        "ty.showSyntaxErrors": false
+    }
+    ```
```

Comment:
I don't think this has been implemented yet. Are you going to open a PR in https://github.com/astral-sh/ty-vscode ?

Reply:
See https://github.com/astral-sh/ty-vscode/pull/264